app/Helpers/database/ticket.php

```diff
@@ -413,22 +415,14 @@ function countRequestTicketsByUser(string $user): int
         return 0;
     }
 
-    sanitize_sql_inputs($dev);
+    $count = Ticket::whereHas('achievement', function ($query) use ($user) {
+        $query->where('Author', $user)
+            ->whereIn('Flags', [AchievementFlag::OfficialCore, AchievementFlag::Unofficial]);
+    })
+    ->where('ReportState', TicketState::Request)
+    ->count();
 
-    $query = "
-        SELECT count(*) as count
-        FROM Ticket AS tick
-        LEFT JOIN Achievements AS ach ON ach.ID = tick.AchievementID
-        LEFT JOIN UserAccounts AS ua ON ua.ID = tick.ReportedByUserID
-        WHERE ua.User = '$user' AND ach.Flags IN (3, 5) AND tick.ReportState = " . TicketState::Request;
-
-    $dbResult = s_mysql_query($query);
-
-    if (!$dbResult) {
-        return 0;
-    }
-
-    return (int) mysqli_fetch_assoc($dbResult)['count'];
+    return $count;
```

Comment:
(indendation)
minor: I'd make this a direct return of the count without the local variable. we have that in some places but personally prefer to not use local variables when they do not add context for better understanding.